content/blog/asyncapi-github-actions.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "GitHub Actions for AsyncAPI aka Automate your workflows related to AsyncAPI documents"
```

Comment:
Isn't that title too looooong? 😄

Reply:
maybe, but I don't see it as a problem really, at least you know what it is

Reply:
Fair enough but give it another thought before deciding to leave it as it is.